src/Pages/userList.tsx

```diff
@@ -36,10 +44,25 @@ export const UserListPage = () => {
     false
   );
   const [viewBlockedUsers, setViewBlockedUsers] = useState<boolean>(false);
+  const [getTechsTrigger, setTechsTrigger] = React.useState(true);
+  const [newTechName, setTechName] = React.useState<string>("Generic Name");
+  const [newTechPassword, setTechPassword] = React.useState<string>(
+    "Generic Password"
+  );
+  const [openDeleteTechConfirmPopUp, setDeleteConfirmPopUp] = useState<boolean>(
+    false
+  );
+  const [openAddTechConfirmPopUp, setAddTechConfirmPopUp] = useState<boolean>(
+    false
+  );
   useEffect(() => {
     !viewBlockedUsers
-      ? getActiveUsers().then((users) => {
-          setUserList(users.users || []);
+      ? getUsers().then((r) => {
```

Comment:
a co to za revert kodu :P niech tu będzie nadal `getActiveUsers`

Reply:
Oksy